### ISSUE TITLE
Very interesting. 

### ISSUE THREAD

**Issue by tynor88** [segator/PlexRemoteTranscoderDocker#2]
Hi,

This is very interesting. I will be getting my new unRAID server on tuesday, and i will definitely try this out with scaleway! Hoping for further development.

Cheers :)

**Comment by segator** [comment]
Yes if I see the people really need other cloud computing providers I will add it,
I'm using with DigitalOcean because it's the most faster to start the VM's.
Other Idea it would be port all this system to ARM dockers, to allow rpi1/2 use full plex :)
